src/library_glemu.js

```diff
@@ -394,7 +402,7 @@ var LibraryGLEmulation = {
         if (GL.stringCache[name_]) return GL.stringCache[name_];
         switch (name_) {
           case 0x1F03 /* GL_EXTENSIONS */: // Add various extensions that we can support
-            var ret = stringToNewUTF8((GLctx.getSupportedExtensions() || []).join(' ') +
```

Comment:
Since all of the "fake" extensions are GL_ prefixed I'm guessing that here we would want to get GL_ prefixed versions of WebGL extensions (if enabled) too. Though I see that doesn't seem to have been true before.

Reply:
Agreed.  Perhaps that should be a followup though, since it seems like a separate change/fix.